src/components/radio-button/radio-button.jsx

```diff
@@ -267,7 +270,8 @@ const RadioButtonComponent = (
 		<label
 			className={ cn(
 				'inline-flex items-center relative cursor-pointer transition-all duration-300',
-				!! label && 'items-start justify-between min-w-[180px] ',
+				!! label && 'items-start justify-between',
+				minWidth && 'min-w-[180px]',
```

Comment:
Can you please add the color to the Tailwind config and then use it here instead of an arbitrary class name? If we use an arbitrary class name then it would be hard to override for the developers.

👉 `min-w-[180px]`

Reply:
Color, or width, pls clarify.

Reply:
`180px`.  It Seems the Tailwind merge resolved the arbitrary class name override issue. I checked just a few minutes ago and it's working fine. No need to define it in the config.